### PR TITLE
add `feed` workflow, take 2

### PR DESCRIPTION
**Pull request by ericphanson**
Redo of #43737 but from a branch, not a fork.

### CODE REVIEW

**Comment by DilumAluthge** [comment]
Long term, it will probably be better to use a manifest specific for the `feed` workflow? Just so that if things break, it's easy to roll back to the last good manifest?

Doesn't need to be done in this PR.

**Review by DilumAluthge — CHANGES_REQUESTED** [review]
Can you add a conditional that only runs this action if the PR is not made from a fork?

**Comment by DilumAluthge** [comment]
@logankilpatrick would be good for you to double check the code.

**Review by DilumAluthge — APPROVED** [review]
This should have been an approval. LGTM as long as you add the "don't run on forks" logic before merging.

**Comment by ericphanson** [comment]
> Can you add a conditional that only runs this action if the PR is not made from a fork?

I'm not sure how to do this :(. I added a julia-side conditional to bail if the endpoint is empty, but I think ideally it would be earlier in the workflow like where I check if the label is the right one.

**Review by logankilpatrick — APPROVED** [review]
Approved once the endpoint secret is updated.

**Comment by logankilpatrick** [comment]
Ideally Slack.jl would check this, if you add a PR I can accept that

**Comment by DilumAluthge** [comment]
> > Can you add a conditional that only runs this action if the PR is not made from a fork?
> 
> I'm not sure how to do this :(. I added a julia-side conditional to bail if the endpoint is empty, but I think ideally it would be earlier in the workflow like where I check if the label is the right one.

This conditional gives you "PR is not from a fork":
> if: github.event_name == 'pull_request' && github.repository == github.event.pull_request.head.repo.full_name

Stick that in your existing `if` with an `&&` in between.

**Comment by DilumAluthge** [comment]
Secret updated.

**Comment by DilumAluthge** [comment]
You can take the Julia-side conditional out once you've assembled the massive unholy YAML conditional.

**Comment by ericphanson** [comment]
> You can take the Julia-side conditional out once you've assembled the massive unholy YAML conditional.

😄. I took out the `github.event_name == 'pull_request'` since we only run on PRs, and left in the Julia-side one but updated the language to not mention forks. I thought it was still helpful since if we get the secret's name wrong it will tell us more clearly than those giant webpage dumps we were getting in the other PR.

File: .github/workflows/feed.yml
Comment (DilumAluthge) [inline .github/workflows/feed.yml:18]:
```suggestion
        if isempty(endpoint)
```

**Pull request merged**
